compose/k8s/mandatory/config-secret.yaml

```diff
@@ -9,3 +9,5 @@ data:
   DJANGO_SECRET_KEY: {{ '${DJANGO_SECRET_KEY}' | b64encode }}
   DJANGO_MEDIA_ROOT: {{ '${DJANGO_MEDIA_ROOT}' | b64encode }}
   DJANGO_DEBUG: {{ '${DJANGO_DEBUG}' | b64encode }}
+  SENTRY_DSN: {{ '${SENTRY_DSN}' | b64encode }}
```

Comment:
do you need to decode the value on the other end?

Reply:
No. I don't know why we need to follow this convention vs allow k8s to do this for us.